docs/book/advanced-guide/getting_started_on_gcp.md

```diff
@@ -0,0 +1,293 @@
+# Getting started with Google Cloud
+
+This guide will take you from the default local stack to a basic cloud 
```

Comment:
Ok the only thing missing now IMO is the motivation of this section. We need to make clear what this guide is about. We can say something like "To get started using ZenML on the cloud, you need some basic infrastructure up and running that you can then make more complicated depending on your use-case. This guide sets up the easiest MLOPs stack that we can run on GCP with ZenML" or osmething?

P.S. I would call the title of this doc only `GCP` for simplicity